projects/packages/blaze/src/class-dashboard-rest-controller.php

```diff
@@ -163,6 +163,17 @@ public function register_rest_routes() {
 			)
 		);
 
+		// WordAds DSP API Site Stats routes
+		register_rest_route(
+			static::$namespace,
+			sprintf( '/sites/%1$d/wordads/dsp/api/v1.1/sites/%1$d/stats(?P<sub_path>[a-zA-Z0-9-_\/]*)(\?.*)?', $site_id ),
```

Comment:
Could you make the API version dynamic? Like we did for the campaign's endpoint?
```
(?P<api_version>v[0-9]+\.?[0-9]*)
```

I think it will make it easier in case we need to have a version 1.0 for this specific endpoint.

But, its not a blocking comment, we currently don't have a v1.0 stats base controller.

Reply:
Good shout, will do!